packages/common/src/utils/isChainSupportedByWallet.ts

```diff
@@ -0,0 +1,53 @@
+import { NetworkVMType } from '@avalabs/core-chains-sdk';
+import { SecretType } from '@core/types';
+
+export const isChainSupportedByWallet = (
+  networkType: NetworkVMType,
+  secretType?: SecretType,
+) => {
+  if (!networkType || !secretType) {
+    return false;
+  }
+
+  switch (networkType) {
+    case NetworkVMType.EVM:
+      return true;
+    case NetworkVMType.AVM:
+    case NetworkVMType.PVM:
+    case NetworkVMType.CoreEth:
+      return (
+        secretType === SecretType.Mnemonic ||
```

Comment:
This is just a nice to have, but how about instead of writing it out like this, using some functions? 😅 
```typescript
case NetworkVMType.AVM:
case NetworkVMType.PVM:
case NetworkVMType.CoreEth:
  return [
    SecretType.Mnemonic,
    SecretType.Seedless,
    SecretType.PrivateKey,
    SecretType.Ledger,
    SecretType.LedgerLive,
    SecretType.Keystone3Pro,
  ].includes(secretType);
```
\#niceToHave

Reply:
I'll smuggle it in some future PR 🍻